test/src/test/kotlin/graphql/nadel/tests/next/fixtures/introspection/NamespaceLikeShortCircuitsIntrospectionTestSnapshot.kt

```diff
@@ -0,0 +1,50 @@
+// @formatter:off
+package graphql.nadel.tests.next.fixtures.introspection
+
+import graphql.nadel.tests.next.ExpectedNadelResult
+import graphql.nadel.tests.next.ExpectedServiceCall
+import graphql.nadel.tests.next.TestSnapshot
+import graphql.nadel.tests.next.listOfJsonStrings
+import kotlin.Suppress
+import kotlin.collections.List
+import kotlin.collections.listOf
+
+private suspend fun main() {
+    graphql.nadel.tests.next.update<NamespaceLikeShortCircuitsIntrospectionTest>()
+}
+
+/**
+ * This class is generated. Do NOT modify.
+ *
+ * Refer to [graphql.nadel.tests.next.UpdateTestSnapshots]
+ */
+@Suppress("unused")
+public class NamespaceLikeShortCircuitsIntrospectionTestSnapshot : TestSnapshot() {
+    override val calls: List<ExpectedServiceCall> = listOf(
+            )
+
+    /**
+     * ```json
+     * {
+     *   "data": {
+     *     "test": {
+     *       "__typename": "TestQuery"
+     *     }
+     *   }
+     * }
+     * ```
+     */
+    override val result: ExpectedNadelResult = ExpectedNadelResult(
+            result = """
+            | {
+            |   "data": {
+            |     "test": {
+            |       "__typename": "TestQuery"
```

Comment:
Prior to this PR's fix, it would error out like so

```json
{
  "errors": [
    {
      "message": "Exception while fetching data (/test) : Internal error: should never happen: This data fetcher should NEVER be called from Nadel",
      "locations": [
        {
          "column": 2,
          "line": 1
        }
      ],
      "path": [
        "test"
      ],
      "extensions": {
        "classification": "DataFetchingException"
      }
    }
  ],
  "data": {
    "test": null
  }
}
```